listen.gemspec

```diff
@@ -33,6 +33,9 @@ Gem::Specification.new do |s|
   s.add_dependency 'rb-fsevent', '>= 0.9.3'
   s.add_dependency 'rb-inotify', '>= 0.9.7'
 
+  # Used to show warnings at runtime
+  s.add_dependency 'ruby_dep', '~> 1.1'
```

Comment:
Yes - that was the intention.

It's a bit tricky, but the gem is needed in 3 places:

1. During runtime, so warnings about Ruby version can be shown (`add_dependency`)
2. During parsing of the `listen.gemspec` (with Bundler - that's why the exception handling is in place in the gemspec).
3. When building the gem (but that's handled by Bundler).